lib/node_modules/@stdlib/iter/cartesian-square/README.md

```diff
@@ -0,0 +1,133 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# iterCartesianSquare
+
+> Create an iterator which generates the Cartesian square of an input array-like object.
+
+<section class="intro">
+
+</section>
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var iterCartesianSquare = require( '@stdlib/iter/cartesian-square' );
+```
+
+#### iterCartesianSquare( x )
+
+Returns an iterator which generates the Cartesian Square of an input array-like object.
+
+```javascript
+var x = ['a', 'b', 'c'];
+var pair;
+var cartesianSquare = iterCartesianSquare( x, n );
+
+for ( pair of cartesianSquare ) {
```

Comment:
ES5. Just demonstrate manually returning the first few values. See other iterator packages.